ide/editor.autosave/i18n/omegat-ja/source/javahelp/org/netbeans/modules/autosave/docs/autosave-about.html

```diff
@@ -0,0 +1,38 @@
+<!--
+
+  -- Licensed to the Apache Software Foundation (ASF) under one
+  -- or more contributor license agreements.  See the NOTICE file
+  -- distributed with this work for additional information
+  -- regarding copyright ownership.  The ASF licenses this file
+  -- to you under the Apache License, Version 2.0 (the
+  -- "License"); you may not use this file except in compliance
+  -- with the License.  You may obtain a copy of the License at
+
+  -- http://www.apache.org/licenses/LICENSE-2.0
+
+  -- Unless required by applicable law or agreed to in writing,
+  -- software distributed under the License is distributed on an
+  -- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  -- KIND, either express or implied.  See the License for the
+  -- specific language governing permissions and limitations
+  -- under the License.
+
+-->
+<html>
+    <head>
+        <title>About AutoSave Module</title>
+        <link rel="stylesheet" href="nbdocs:/org/netbeans/modules/usersguide/ide.css" type="text/css">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    </head>
+    <body>
+        <h2>About AutoSave Module</h2>
+         <p>
+            This module provides the functionality to automatically save all yours unsaved opened files.
```

Comment:
This string is repeated 11 times in this commit--are these generated files? Each internationalized string like this should only occur in one place.